protoc-builder/Makefile

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright 2022 The Sigstore Authors.
```

Comment:
Wrong copyright year? Not sure why the linter didn't see this

Reply:
yeah I was wondering the same thing about addlicense, it seems to not actually check that.